index.html

```diff
@@ -0,0 +1,357 @@
+            <div class="review">
+              <span>1,795 Comments</span>
+              <span class="icon">
+                <img src="./resources/sorting.svg" alt="options" /> </span
+              ><span> SORT BY</span>
+            </div>
+            <div class="commentLists">
+              <div class="singleComment">
+                <div class="icon">
+                  <img src="./resources/userSpeaking.svg" alt="options" />
+                </div>
+                <div class="newComment"">
+                <input
+                  type="text"
+                  name="newComment"
+                  placeholder="Add a public comment..."
+                />
+                <div class="buttons">
+                  <button type="submit">CANCEL</button>
+                  <button type="submit">COMMENT</button>
+                </div>
+              </div>
+            </div>
+            <div class="singleComment">
+              <!-- <div class="icon">
+                <img src="./resources/coment-user1.svg" alt="options" />
+              </div>              <div class="icon"> -->
+                <img src="./resources/coment-user1.svg" alt="options" />
+              </div>
+              <div>
+                <p class="name">Tadesse Alemayehu</p>
+                <p class="comment">
+                  this is the comment made by the commenter. this is the comment
+                  made by the com by the commenter. this is the comment made by
+                  the commenter this is the comment made by the commenter.
+                </p>
+              </div>
+            </div>
+            <div class="singleComment">
+              <div class="icon">
+                <img src="./resources/coment-user2.svg" alt="options" />
+              </div>
+              <div>
+                <p class="name">Bekalu berie</p>
+                <p class="comment">
+                  this is the comment made by the commenter name this is the
+                  comment made by the commenter name this is the comment made by
+                  the commenter name
+                </p>
+              </div>
+            </div>
+            <div class="singleComment">
+              <div class="icon">
+                <img src="./resources/coment-user3.svg" alt="options" />
+              </div>
+              <div>
+                <p class="name">maru sisay</p>
+                <p class="comment">
+                  this is the comment made by the commenter.
+                </p>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="sideBar">
+      <div class="top">
+        <button>All</button>
+        <button>Music of Ethiopian</button>
+        <button>Related</button>
+        <button>Recent uploaded</button>
+      </div>
+      <div class="SinglePreview">
+        <div class="img">
+          <img src="./resources/thumbnail/preview1.png" alt="" />
```

Comment:
- [ ] What do you think about adding the `alt` to all the `img` tag? It provides alternative information for the image if a user for some reason cannot view the image.